pkgs/by-name/ch/chart-releaser/package.nix

```diff
@@ -0,0 +1,67 @@
+{ buildGoModule
+, coreutils
+, fetchFromGitHub
+, git
+, installShellFiles
+, kubernetes-helm
+, lib
+, makeWrapper
+}:
+
+buildGoModule rec {
+  pname = "chart-releaser";
+  version = "1.6.1";
+
+  # 0102fa30dd78df0d0f8093c4607e6080972ff82d
```

Comment:
This some kind of checksum? Do we need this comment here, is it useful?

Reply:
The comment is not needed. Sorry! I should have deleted it.